client/rpc-spec-v2/src/chain_head/chain_head_storage.rs

```diff
@@ -81,17 +96,15 @@ where
 
 		result
 			.map(|opt| {
-				opt.map(|storage_data| {
-					QueryResult::Ok(StorageResult::<String> {
-						key: hex_string(&key.0),
-						result: StorageResultType::Value(hex_string(&storage_data.0)),
-					})
-				})
+				QueryResult::Ok(opt.map(|storage_data| StorageResult::<String> {
```

Comment:
Nit: I personally would just use  `Ok` instead of `QueryResult::Ok`, but I guess that is just personal preference.

Reply:
(It might be that `QueryResult::Ok` is used here to convey the type information that comes along with `QueryResult` :))